examples/distributed-key-value-store/src/main.rs

```diff
@@ -22,6 +22,7 @@
 
 use async_std::io;
 use futures::{prelude::*, select};
+use libp2p::bytes::Bytes;
```

Comment:
Why are we re-exporting this? We should just depend on it directly. `bytes` is already > 1.0.

Reply:
> Why are we re-exporting this?

I'm not sure, i found it was exported so thought it was perhaps a preference. I'm fine eitherway here, so I'll update.